static/app/views/insights/browser/webVitals/queries/useWebVitalsIssuesQuery.tsx

```diff
@@ -1,26 +1,37 @@
 import {normalizeDateTimeParams} from 'sentry/components/organizations/pageFilters/parse';
 import type {Group, ISSUE_TYPE_TO_ISSUE_TITLE} from 'sentry/types/group';
+import {defined} from 'sentry/utils';
 import {useApiQuery} from 'sentry/utils/queryClient';
 import useOrganization from 'sentry/utils/useOrganization';
 import usePageFilters from 'sentry/utils/usePageFilters';
 
-export function getIssueQueryFilter(
-  performanceIssues: Array<keyof typeof ISSUE_TYPE_TO_ISSUE_TITLE>
-) {
-  return `is:unresolved issue.type:[${performanceIssues?.join(',')}]`;
+export function getIssueQueryFilter({
+  issueTypes,
+  transaction,
+}: {
+  issueTypes: Array<keyof typeof ISSUE_TYPE_TO_ISSUE_TITLE>;
+  transaction?: string;
+}) {
+  return `is:unresolved issue.type:[${issueTypes?.join(',')}]${defined(transaction) ? ` transaction:${transaction}` : ''}`;
 }
 
-export function useWebVitalsIssuesQuery(
-  performanceIssues: Array<keyof typeof ISSUE_TYPE_TO_ISSUE_TITLE>
-) {
+export function useWebVitalsIssuesQuery({
+  issueTypes,
+  transaction,
+  enabled = true,
+}: {
+  issueTypes: Array<keyof typeof ISSUE_TYPE_TO_ISSUE_TITLE>;
+  enabled?: boolean;
+  transaction?: string;
+}) {
   const organization = useOrganization();
   const {selection} = usePageFilters();
   return useApiQuery<Group[]>(
```

Comment:
Is there a hook we can use to fetch issues? iirc there actually isn't, which to me is extremely surprising considering issues is our bread and butter